back/engines/commercial/bulk_import_ideas/app/services/bulk_import_ideas/import_ideas_service.rb

```diff
@@ -155,7 +156,7 @@ def add_published_at(idea_row, idea_attributes)
 
       published_at = nil
       begin
-        published_at = Date.parse idea_row[:published_at]
+        published_at = Date.iso8601 idea_row[:published_at]
```

Comment:
Date.iso8601 does not accept the date format that we document and use ("DD-MM-YYYY"):

```
Date.iso8601 '03-02-2022'
Date::Error: invalid date
```
It accepts "YYYY-MM-DD":
```

Date.iso8601 '2022-02-03'
=> Thu, 03 Feb 2022
```
So I think this change is not correct. When we discussed this in the team, we decided not to use format "YYYY-MM-DD", but keep it for a future major upgrade of the import tool, because people are used to the current format "DD-MM-YYYY".

Reply:
Good catch, I used the wrong formats across all the bulk idea import work. Fixing it now.